new Widget "widget_verkehrsinfo.js"

paul179 opened the pull request:
please add new Widget for Modul Verkehrsinfo

paul179 committed:
new Widget "widget_verkehrsinfo.js"

please add new Widget for Modul Verkehrsinfo

knowthelist commented:
Thanks for your contribution. But a few things have to be changed, before I can merge this new widget.  
- it is always a bad idea to use a sleep function for wasting time. See javascripts setTimeout function to postpone calls. But is it really necessary? 
- update function :  dev can not be "*". Why this if part?

paul179 committed:
Update widget_verkehrsinfo.js

Hello,

> •it is always a bad idea to use a sleep function for wasting time. See javascripts setTimeout function to postpone calls. But is it really necessary?

please see
https://forum.fhem.de/index.php/topic,55118.msg544555.html#msg544555 and I have this delete

> •update function : dev can not be "*". Why this if part?
for Problem see link I have this copy from widget calview, I have this changed.

paul179 commented:
Hello,

> •it is always a bad idea to use a sleep function for wasting time. See javascripts setTimeout function to postpone calls. But is it really necessary?

please see
https://forum.fhem.de/index.php/topic,55118.msg544555.html#msg544555 and I have this delete

> •update function : dev can not be "*". Why this if part?
for Problem see link I have this copy from widget calview, I have this changed.

knowthelist commented:
Normally a delay is not necessary. There must be a conceptual failure. Via long poll for each reading updated in FHEM, the update function get called.

paul179 commented:
I understand what I should do? I have the Code changed!!

paul179 committed:
Update widget_verkehrsinfo.js

paul179 committed:
Update widget_verkehrsinfo.js

paul179 committed:
Update widget_verkehrsinfo.js
